src/main/resources/templates/cypher/hintlist.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
Tento soubor by se mel jmenovat src/main/resources/templates/cypher/detail.html Protoze to ukazuje detail sifry i s jejimy hinty.

Reply:
Done